src/deps.ts

```diff
@@ -1 +1,2 @@
 export { parse } from "https://deno.land/std@0.138.0/flags/mod.ts";
+export * as path from "https://deno.land/std@0.134.0/path/mod.ts";
```

Comment:
Random thought, don't have to do it now, but as we add stuff to `deps.ts`, the naming of the exports in this file start to matter. `parse` on its own now doesn't provide much context (parsing of what? URLs? flags?). I think we'll want to start naming these exports a bit more accurately. Again, don't need to do this now, but I think the next time we add something else to this file (rule of threes!), we should tweak the export names.

Reply:
This is one of the things I don't like that come about from following the `deps.ts` standard😅